dozer/sources/RSSSources.py

```diff
@@ -0,0 +1,240 @@
+        return f"New Post from {self.full_name} from {data['author']}:\n" \
+               f"{data['title']}\n" \
+               f">>> {data['description']}\n" \
+               f"Read more at {data['url']}"
+
+
+class FRCBlogPosts(RSSSource):
+    """Official blog posts from the FIRST Robotics Competition"""
+    url = "https://www.firstinspires.org/robotics/frc/blog-rss"
+    base_url = "https://www.firstinspires.org/robotics/frc/blog/"
+    full_name = "FRC Blog Posts"
+    short_name = "frc"
+    description = "Official blog posts from the FIRST Robotics Competition"
+    color = discord.colour.Color.dark_blue()
+
+
+class TBABlog(RSSSource):
+    """Posts from The Blue Alliance's Blog"""
+    url = "https://blog.thebluealliance.com/feed/"
+    base_url = "https://blog.thebluealliance.com/"
+    full_name = "The Blue Alliance Blog"
+    short_name = "tba-blog"
+    description = "Posts from The Blue Alliance's Blog"
+    color = discord.colour.Color.dark_blue()
+
+
+class CDLatest(RSSSource):
+    """Official blog posts from the FIRST Robotics Competition"""
+    url = "https://www.chiefdelphi.com/latest.rss"
+    base_url = "https://www.chiefdelphi.com/latest"
+    full_name = "Chief Delphi"
+    short_name = "cd"
+    description = "Official blog posts from the FIRST Robotics Competition"
+    color = discord.colour.Color.orange()
+
+
+class FRCQA(RSSSource):
+    """Answers from the official FIRST Robotics Competition Q&A system"""
+    url = "https://frc-qa.firstinspires.org/rss/answers.rss"
+    base_url = "https://frc-qa.firstinspires.org/"
+    full_name = "FRC Q&A Answers"
+    short_name = "frc-qa"
+    description = "Answers from the official FIRST Robotics Competition Q&A system"
+    color = discord.colour.Color.dark_blue()
+
+
+class FTCBlogPosts(RSSSource):
+    """The official FTC Blogspot blog"""
+    url = "http://firsttechchallenge.blogspot.com//feeds/posts/default"
+    base_url = "http://firsttechchallenge.blogspot.com/"
+    full_name = "FTC Blog Posts"
+    short_name = "ftc"
+    description = "The official FTC Blogspot blog"
+    color = discord.colour.Color.orange()
+
+
+class FTCForum(RSSSource):
+    """The official FTC Forum posts"""
+    url = "https://ftcforum.firstinspires.org/external?type=rss2&nodeid=1"
+    base_url = "https://ftcforum.firstinspires.org/"
+    full_name = "FTC Forum Posts"
+    short_name = "ftcforum"
+    description = "The official FTC Forum Posts"
+    color = discord.colour.Color.orange()
+
+
+class JVNBlog(RSSSource):
+    """Blog posts by John V Neun, 148 Head Engineer"""
+    url = "https://johnvneun.com/blog?format=rss"
+    base_url = "https://johnvneun.com/"
+    full_name = "JVN's Blog"
+    short_name = "jvn"
+    aliases = ['148', 'robowranglers']
+    description = "Blog posts by John V Neun, 148 Head Engineer"
+    color = discord.colour.Color(value=000000)
+    disabled = True
+    # For some reason, the XML parser hates this source. Investigate later.
```

Comment:
It looks like since JVN's blog uses HTTP/2.0, the keep-alive parameter is required, but when it's added, it only recieves part of the response, and since the RSS for the site is fairly large, this fails to parse it (invalid XML). It looks like the issue is with aiohttp instead of the XML parser. I still don't have a fix.